Makefile

```diff
@@ -30,7 +30,7 @@ clean:
 	rm -rf $(TARGET)
 
 image: build
-	docker build -t $(REGISTRY)/$(TARGET):latest .
-
+	docker build -t $(REGISTRY)/$(TARGET):latest . && \
+        docker tag $(REGISTRY)/$(TARGET):latest $(REGISTRY)/$(TARGET):${VERSION}
```

Comment:
why not build directly with `${VERSION}` tag. ?

Reply:
I assumed it is useful to have the latest tag locally. Let me get rid of it.